### PR TITLE
Setup genesis

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

File: cli/src/genesis.rs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    println!("STEP 2");
+    node.progress_for_consensus().await.unwrap();
+    println!("PRESS ENTER TO RUN SERVER -------- [C]");
+    get_input();
+    println!("SERVE STARTED -------- [C]");
+    let mut node = node.serve().await.unwrap();
+    println!("SERVE FINISHED");
+
+    println!("STEP 6");
```
Comment (YoonhoKim0527) [inline cli/src/genesis.rs:57]: Why is this "STEP 6" ?

File: cli/src/genesis.rs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    println!("SERVE STARTED -------- [D]");
+    let node = node.serve().await.unwrap();
+    println!("SERVE FINISHED");
+
+    let result = node.get_consensus_status().await.unwrap();
+    println!("{:?}", result);
+}
+
+pub async fn run_genesis_non_proposer(config: Config, path: &str) {
+    println!("STEP 0");
```
Comment (YoonhoKim0527) [inline cli/src/genesis.rs:70]: I can't understand this "STEP 0" usage. If there is only one step, then why should we print "STEP 0" ?

File: cli/Cargo.toml [hunk 3]
```diff
@@ -19,3 +19,5 @@ semver = "1.0.0"
 clap = { version = "4.0", features = ["derive"] }
 rand = "0.8.5"
 hex = "0.4.3"
+env_logger = "0.10.0"
+color-eyre = "0.6.2"
```
Comment (TomTaehoonKim) [inline cli/Cargo.toml:23]:
```suggestion
color-eyre = "0.6.2"

```

File: cli/src/genesis.rs [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    std::io::stdin().read_line(&mut _buf).unwrap();
+}
+
+fn get_commit_hash() -> CommitHash {
+    loop {
+        let mut buf = String::new();
+        io::stdin().lock().read_line(&mut buf).unwrap();
+        let x = buf.trim().to_owned();
+        if x.len() != 40 {
+            println!("INVALID LENGTH: {}", x.len());
```
Comment (lemonholic) [inline cli/src/genesis.rs:17]:
```suggestion
            println!("INVALID LENGTH (PROVIDE 40): {}", x.len());
```

**Pull request merged**